docs/cloud.md

```diff
@@ -0,0 +1,234 @@
+# Feldera Cloud
+
+These instructions explain how to run the Feldera Platform on a AWS Elastic Kubernetes Service (EKS). 
+
+## Prerequisites
+
+* A dedicated AWS account (e.g. created using AWS organizations).
+* [AWS CLI](https://aws.amazon.com/cli/)
+* [eksctl](https://github.com/weaveworks/eksctl): to bring up and configure AWS Elastic Kubernetes Service.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/): to interact with the deployed EKS cluster.
+* [Helm](https://helm.sh/docs/intro/quickstart/): to deploy and upgrade Feldera Cloud on EKS.
+* A Feldera license, as part of which you will receive a Helm Chart and container artifacts to run Feldera Cloud.
+
+## Deployment overview
+
+We will go through the following steps to deploy Feldera Cloud on EKS:
+
+* Use `eksctl` to create a dedicated VPC and bring up an EKS cluster inside it.
+* Use `Helm` to deploy Feldera Cloud inside the EKS cluster.
+* Configure input and output data sources on EKS.
+* Connect these data sources to Feldera.
+
+
+## Create an EKS cluster
+
+Make sure your `aws` CLI is configured to work with your AWS account. If you haven't done so already,
+run `aws configure` and enter the required credentials.
+
+Next, create a configuration file named `eks-config.yaml` for running `eksctl`.
+
+
+```
+# File name: eks-config.yaml
+
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: feldera-eks
+  region: us-west-1
+  version: "1.27"
+
+# OIDC provider is needed to use EBS
+iam:
+  withOIDC: true
+
+managedNodeGroups:
+  - name: managed-ng
```

Comment:
Lets add some explicit `volumeSize` and `additionalVolumes`. This explicit mounts can be used by RocksDB, RedPanda, PostgreSQL storage. https://eksctl.io/usage/nodegroup-additional-volume-mappings/

```
  - name: managed-ng
    desiredCapacity: 3
    instanceType: m5.large
    privateNetworking: true
    volumeSize: 80
    additionalVolumes:
      - volumeName: '/mnt/storage1' # required
        volumeSize: 200
        volumeType: 'gp3'
````

Reply:
It's better to update this after @gz @blp can give us guidance on the instance types and storage setup that work well for fault tolerance.